docs/core/tools/index.md

```diff
@@ -81,7 +81,7 @@ When you supply a command to the driver, `dotnet.exe` starts the CLI command exe
 dotnet build
 ```
 
-First, the driver determines the version of the SDK to use. If there is no ['global.json'](global-json.md), the latest version of the SDK available is used. This might be either a preview or stable version, depending on what is latest on the machine.  Once the SDK version is determined, it executes the command.
+First, the driver determines the version of the SDK to use. If there is no [global.json](global-json.md) file, the latest version of the SDK available is used. This might be either a preview or stable version, depending on what is latest on the machine.  Once the SDK version is determined, it executes the command.
```

Comment:
```suggestion
First, the driver determines the version of the SDK to use. If there is no [_global.json_](global-json.md) file, the latest version of the SDK available is used. This might be either a preview or stable version, depending on what is latest on the machine.  Once the SDK version is determined, it executes the command.
```

Reply:
I would normally italicize file names but I avoid applying extra styles to link text. https://docs.microsoft.com/en-us/contribute/text-formatting-guidelines#headings-and-link-text